packages/gitbook/src/components/RootLayout/CustomizationRootLayout.tsx

```diff
@@ -100,6 +100,11 @@ export async function CustomizationRootLayout(props: {
                     <link rel="privacy-policy" href={customization.privacyPolicy.url} />
                 ) : null}
 
+                <meta
```

Comment:
We should not have to do this and instead use https://nextjs.org/docs/app/api-reference/functions/generate-viewport

You should add this to `generateSiteLayoutViewport`